packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx

```diff
@@ -113,7 +141,7 @@ const TopBarSearchModal = () => {
         )}
         <input
           type="text"
-          placeholder={t("search.placeholder")}
+          placeholder={t("Search")}
```

Comment:
Why are you changing this. This is important for the translation team who added this there

Reply:
My bad please. I will revert the change.